### PR TITLE
added parse-decimal-number to definitions

### PR DESCRIPTION
**Pull request by FermiDirak**
added parse-decimal-number to definitions

### CODE REVIEW

File: definitions/npm/parse-decimal-number_v1.x.x/flow_v0.25.x-/parse-decimal-number_v1.x.x.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare module 'parse-decimal-number' {
+  declare module.exports: (value: string, inOptions?: any, enforceGroupSize?: boolean) => number;
```
Comment (pascalduez) [inline definitions/npm/parse-decimal-number_v1.x.x/flow_v0.25.x-/parse-decimal-number_v1.x.x.js]: `inOptions` can be either `string | Array<string> | { thousands: string, decimal: string }`.

Comment (pascalduez) [reply definitions/npm/parse-decimal-number_v1.x.x/flow_v0.25.x-/parse-decimal-number_v1.x.x.js]: The return value can be either `number | NaN`.

Comment (pascalduez) [reply definitions/npm/parse-decimal-number_v1.x.x/flow_v0.25.x-/parse-decimal-number_v1.x.x.js]: Missing `parseDecimalNumber.factorySettings`, `parseDecimalNumber.setOptions`, `parseDecimalNumber.withOptions` signatures.

**Comment by GAntoine** [comment]
@FermiDirak You'll need to add specs for the new def.

**Pull request closed without merging**